contracts/kernel/Kernel.sol

```diff
@@ -168,8 +166,7 @@ contract Kernel is IKernel, KernelStorage, Initializable {
     * @return boolean indicating whether the ACL allows the role or not
     */
     function canPerform(address _entity, address _app, bytes32 _role) constant public returns (bool) {
-        var (isAllowed,) = getPermission(_entity, _app, _role);
-        return isAllowed;
+        return getPermission(_entity, _app, _role);
```

Comment:
Maybe we can rename `getPermission()` to `hasPermission()` and remove `canPerform()` altogether?

Reply:
Good idea